ext/opentelemetry-ext-zipkin/tests/test_zipkin_exporter.py

```diff
@@ -97,10 +97,10 @@ def test_constructor_explicit(self):
     def test_export(self):
 
         span_names = ("test1", "test2", "test3", "test4")
-        trace_id = 0x6E0C63257DE34C926F9EFCD03927272E
-        span_id = 0x34BF92DEEFC58C92
+        trace_id = 0x0E6C63257DE34C926F9EFCD03927272E
```

Comment:
IMO it would be better if we added a specific test to ensure that zero-padding works as intended. The current test makes no indicator that the zero-padded traceid is intentional to catch bugs.

A good test with a docstring per regression to educate and ensure future refactors are aware of the rationale for the test, is great.

Reply:
Thanks for the review, will do your suggestions tomorrow.
I hope I can get the CLA signed by my company though. 
It's weird, I had the approved CLA for CNCF last month but this got switched to EasyCLA which requires a new signature.

Reply:
Thank you! I believe if you signed at least one of the CLAs, that's ok for us to merge. But I'll let the maintainers weigh in there.